crates/ruma-events/src/media_preview_config.rs

```diff
@@ -9,7 +9,7 @@ use crate::{macros::EventContent, PrivOwnedStr};
 
 /// The content of an `m.media_preview_config` event.
 #[cfg(feature = "unstable-msc4278")]
```

Comment:
If realized that we don't need all these cargo feature flags, because the whole module is behind a feature flag.